src/swarm/neo/request/RequestEventDispatcher.d

```diff
@@ -1010,3 +1012,43 @@ public struct RequestEventDispatcher
         fiber.resume(this.token);
     }
 }
+
+/*******************************************************************************
+
+    Ocean's ocean.core.Mutation.removeShift adapted to work on AcquiredBuffersOf
```

Comment:
Do you mean adapted to work on `VoidBufferAsArrayOf`? (Maybe I'm misunderstanding.)

Reply:
Yes, typo :+1: